### PR TITLE
Cleanup code in socket.rs

### PR DESCRIPTION
**Pull request by jyn514**
This can just use `?` instead of an explicit match.

### CODE REVIEW

File: src/commands/dev/socket.rs [hunk 1]
```diff
@@ -115,44 +115,40 @@ async fn print_ws_messages(
...
-                        print_json(json_parse, event.to_string());
-                    }
-                    _ => {}
-                }
+        let message = message?;
+        let message_text = message.into_text().unwrap();
+        log::info!("{}", &message_text);
+
+        let parsed_message: Result<protocol::Runtime> = serde_json::from_str(&message_text)
+            .map_err(|e| anyhow!("Failed to parse event:\n{}", e));
```
Comment (jyn514) [inline src/commands/dev/socket.rs:123]: I just noticed we never actually handle this error - it's silently discarded in the `match`. Should it use `?` instead?
```suggestion
        let parsed_message: Result<protocol::Runtime> = serde_json::from_str(&message_text)
            .map_err(|e| anyhow!("Failed to parse event:\n{}", e))?;
```

Comment (jspspike) [reply src/commands/dev/socket.rs:123]: I'm not sure if that would work, if the message failed to parse than it's likely a heartbeat message and therefore we just want to noop and move on. If we return the Err out of the function the `try_join!` on line 61 might kill everything but if you try and it and it doesn't stop dev after receiving heartbeat messages then I think it's fine.

Comment (jyn514) [reply src/commands/dev/socket.rs:123]: Ok, we should probably at least log it then. Right now the `anyhow!` doesn't do anything, it just allocates and then immediately deallocates.

Comment (jspspike) [reply src/commands/dev/socket.rs:123]: Yeah that might be a good idea, we have that `log::info` on line 121 but maybe a debug for the result if it failed to parse

Comment (jyn514) [reply src/commands/dev/socket.rs:123]: Great, added that :)

**Review by jspspike — APPROVED** [review]
Looks good to me, tested and ran into no issues

**Pull request merged**